tests/components/notify/test_command_line.py

```diff
@@ -31,12 +31,14 @@ def test_setup(self):
 
     def test_bad_config(self):
         """Test set up the platform with bad/missing configuration."""
-        self.assertFalse(setup_component(self.hass, notify.DOMAIN, {
-            'notify': {
-                'name': 'test',
-                'platform': 'bad_platform',
-            }
-        }))
+        # Platform should not be set up, but component should be set up.
+        with assert_setup_component(0):
+            self.assertTrue(setup_component(self.hass, notify.DOMAIN, {
```

Comment:
With the change to notify component this test doesn't add value anymore, I think. The notify component will be set up even with bad config, due to config validation removing the bad parts.